src/twisted/web/test/test_newclient.py

```diff
@@ -962,6 +961,32 @@ def test_ignored1XXResponseCausesLog(self):
         self.assertEquals(event["log_format"], "Ignoring unexpected {code} response")
         self.assertEquals(event["code"], 103)
 
+    def test_headerShortEnough(self):
+        """Test that HTTP responses with header lines up to 65536 bytes long
+        are allowed."""
```

Comment:
Maybe keep it simple :)

based on https://jml.io/test-docstrings/

```suggestion
        """
        It allows receiving HTTP header lines up to 65536 bytes long.
        """
```